server/program/program.model.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
-const { Model } = require('sequelize');
+const { Model, Sequelize } = require('sequelize');
+const trim = require('lodash/trim');
 
+const { Op } = Sequelize;
 class Program extends Model {
   static fields(DataTypes) {
     return {
       name: {
         type: DataTypes.STRING,
         allowNull: false,
         unique: true,
-        validate: { notEmpty: true, len: [2, 255] }
+        validate: { notEmpty: true, len: [2, 255] },
+        set(value) { this.setDataValue('name', trim(value)); }
```

Comment:
Ok, but why? You could set default value to make sure it is string and just use native trim? 🤔